### DISCUSSION TITLE
https://nanaeo.cn/about/

### DISCUSSION THREAD

**Discussion by giscus** [MliKiowa/Mlikiowa.github.io#8] [General]
# https://nanaeo.cn/about/

It is a little blog～ - Mlikiowa - Village

https://nanaeo.cn/about/

**Comment by FreeDivers** [comment]
大佬太强了

**Reply by MliKiowa** [reply]
😂没有的 总结一句就是什么也没学到

**Comment by Mitpoppy** [comment]
通过文章了解到是个大学生，毕竟好奇考的什么大学🤣，或者保守说一下是本就是专还是。。。🌝

**Reply by MliKiowa** [reply]
本科( 不过问这个干嘛?

**Reply by Mitpoppy** [reply]
纯属好奇
诶，你回复怎么这么快

**Reply by MliKiowa** [reply]
> 纯属好奇 诶，你回复怎么这么快

因为我正在github上码代码

**Reply by Mitpoppy** [reply]
好家伙，快晚上了还在写代码，你还记得我是谁嘛🌝

**Reply by MliKiowa** [reply]
不太眼熟啊 (

**Reply by Mitpoppy** [reply]
就之前那个用你typecho的附件转github，然后被你加到QQ好友的那个，哈哈

**Reply by MliKiowa** [reply]
> 就之前那个用你typecho的附件转github，然后被你加到QQ好友的那个，哈哈

(主要是不想自己测试 加你们看看你们能不能用

**Reply by Mitpoppy** [reply]
我表示放弃用图床了，正引用你文章的一句话：
> 图床问题: 这个问题我根本不知道怎么解决我还是摆烂吧，博文能不放图我绝对不会放图出来

**Reply by Mitpoppy** [reply]
话说。。。我想知道现在你那个附件插件可以自己改cdn加速链接嘛，之前我看可以在源码改感觉比较麻烦（哈哈我就是懒）
我思路是用cloudflare+github pages自建加速节点，然后文章内的图片链接就是我网站链接/github pages的目录链接，但是不确定可行性

**Reply by MliKiowa** [reply]
> 话说。。。我想知道现在你那个附件插件可以自己改cdn加速链接嘛，之前我看可以在源码改感觉比较麻烦（哈哈我就是懒） 我思路是用cloudflare+github pages自建加速节点，然后文章内的图片链接就是我网站链接/github pages的目录链接，但是不确定可行性

可以的

**Reply by Mitpoppy** [reply]
话说你不是换github存博客了嘛，所以图片的话直接用github做图床不是挺好的嘛🌝

**Reply by MliKiowa** [reply]
> 话说你不是换github存博客了嘛，所以图片的话直接用github做图床不是挺好的嘛🌝

到本地的时候很难受的 混在一起 而且jsd凉了

**Reply by Mitpoppy** [reply]
直接博客链接/目录这样访问图片的链接不就完事了

**Reply by MliKiowa** [reply]
这样后面问题一大堆
